app/Resources/meta/article.yml

```diff
@@ -65,4 +65,7 @@ properties:
     seoMetadata:
         description: "SEO metadata"
         type: Meta
+    place:
+        description: "First place array from place metadata"
```

Comment:
I don't get this description. I think it should be something like: The first place from the article's metadata